pubsub/api/src/subscribe_exactly_once_delivery.php

```diff
@@ -38,6 +38,7 @@ function subscribe_exactly_once_delivery(
 ): void {
     $pubsub = new PubSubClient([
         'projectId' => $projectId,
+        'apiEndpoint' => 'us-west1-pubsub.googleapis.com:443'
```

Comment:
Hmm, maybe a comment to explain what this is for? just something like
```suggestion
        // use the "apiEndpoint" option to set a regional endpoint
        'apiEndpoint' => 'us-west1-pubsub.googleapis.com:443'
```